readme.txt

```diff
@@ -9,11 +9,21 @@ The Gutenberg plugin provides editing, customization, and site building features
 
 == Description ==
 
-“Gutenberg” is a codename for a whole new paradigm for creating with WordPress, that aims to revolutionize the entire publishing experience as much as Gutenberg did the printed word. The project is following a four-phase process that will touch major pieces of WordPress — Editing, Customization, Collaboration, and Multilingual.
+“Gutenberg” is a codename for a whole new paradigm for creating with WordPress, that aims to revolutionize the entire publishing experience as much as <a href="https://en.wikipedia.org/wiki/Johannes_Gutenberg">Johannes Gutenberg</a> did the printed word. The project is <a href="https://wordpress.org/about/roadmap/">following a four-phase process</a> that will touch major pieces of WordPress — Editing, Customization, Collaboration, and Multilingual.
 
-The block editor introduces a modular approach to all parts of your site: each piece of content in the editor, from a paragraph to an image gallery to a headline, is its own block. And just like physical blocks, WordPress blocks can be added, arranged, and rearranged, allowing WordPress users to create media-rich content in a visually intuitive way — and without work-arounds like shortcodes or custom HTML.
+### Who Is This Plugin For?
 
-The block editor first became available in December 2018. We’re always hard at work refining the experience, creating more and better blocks, and laying the groundwork for the future phases of work. Each WordPress release comes ready to go with the stable features from multiple versions of the Gutenberg plugin, so you don’t need to use the plugin to benefit from the work being done here. However, if you’re more adventurous and tech-savvy, the Gutenberg plugin gives you the latest and greatest, so you can join us in testing bleeding-edge features, start playing with blocks, and maybe get inspired to build your own.
+Are you a tech-savvy early adopter who likes testing bleeding-edge and experimental features, and isn’t afraid to tinker with features that are still in active development? Are you interested in contributing toward the Gutenberg open source project? If so, this beta plugin gives you access to the latest features around block and full site editing, as well as a peek into what’s to come.
+
+However, if you are running a production WordPress site, and favor stable, well-tested features, then you might want to stick with the block features already included with WordPress. Gutenberg is not recommended for production websites.
```

Comment:
I think this is a bit heavy handed and that we shouldn't declare what something is or isn't for, since it will depend on the person managing the site. I'd recommend pulling language from here and perhaps linking to this post: https://wordpress.org/news/2021/04/become-an-early-adopter-with-the-gutenberg-plugin/

Reply:
Okay, I've reworked this section to highlight the early access and contributor benefits.